api/.travis.env

```diff
@@ -16,7 +20,7 @@ DB_USERNAME=postgres
 DB_PASSWORD=
 
 CACHE_DRIVER=redis
-# SESSION_DRIVER=memcached
+SESSION_DRIVER=array
```

Comment:
Still needed. The auth manager injects a session driver in the guard instance. We could make our own guard instance and completely eliminate any session references, though as we don't use the session driver inside the current guard instance, it shouldn't affect performance, so I don't think it's necessary to make our own guard from a performance perspective.